staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_test.go

```diff
@@ -225,182 +136,12 @@ func TestGetListNonRecursiveWithConsistentListFromCache(t *testing.T) {
 	storagetesting.RunTestGetListNonRecursive(ctx, t, compactStorage(cacher, server.V3Client), cacher)
```

Comment:
`compactStorage` is a low level construct, maybe in the future the tests that use it could be reworked and moved to a new pkg.